### DISCUSSION TITLE
Support for "graphql-ws" sub protocol

### DISCUSSION THREAD

**Discussion by ManojkumarAlaguduraiEmis** [enisdenjo/graphql-ws#454] [Q&A] [answered]
Hi,

We were using AWS AppSync as our backend server and we are trying to use this package to establish web socket connection. Unfortunately, "graphql-ws" uses "graphql-transport-ws" as the sub-protocol and AWS AppSync users "graphql-ws" as protocol to establish web-socket connection. Is there any plan to support "graphql-ws" protocol?

**Comment by enisdenjo** [comment, marked as answer]
No, the `graphql-ws` protocol is deprecated and its reference implementation [`subscriptions-transport-ws` is archived](https://github.com/apollographql/subscriptions-transport-ws). Please read the [disclaimer in the readme](https://github.com/enisdenjo/graphql-ws#disclaimer).
